hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1683,6 +1683,9 @@ public OzoneFileStatus getFileStatus(OmKeyArgs args) throws IOException {
           volumeName, bucketName, keyName);
       OmKeyInfo fileKeyInfo = metadataManager.getKeyTable().get(fileKeyBytes);
       if (fileKeyInfo != null) {
+        if (args.getRefreshPipeline()) {
```

Comment:
Missing usage of refreshPipeline in listStatus

Reply:
@bharatviswa504 listStatus calls getFileStatus internally. Hence I used it only in getFileStatus.

Reply:
I see that is not the case always, can you once check. Not sure if I am missing something.

Reply:
Yes, you are right. Thanks for the review!